index.html

```diff
@@ -90,20 +92,102 @@ <h3>
+            </div>
+            <div class="first-project">
+              <h3>
+                Setup-Portfolio-Mobile-1st <br>
+                First Week
+              </h3>
+                <ul class="languages-grid">                    
+                        <li class="lang-list l1"><strong>Github</strong></li>
+                        <li class="lang-list l2"><strong>Css3</strong></li>
+                        <li class="lang-list l3"><strong>Git</strong></li>
+                        <li class="lang-list l4"><strong>Html5</strong></li>
+                </ul>
+                <div class="external_link">
+                    <a href="#">See Project</a>
+                </div>
+            </div>
+        </div>
+        <div class="project-2">
+            <div class="first_project_img_div">
+            <img src="img/default_project_img.JPG" alt="first project image" class="first_project_image">
+            </div>
+            <div class="first-project">
+              <h3>
+                Setup-Portfolio-Mobile-1st <br>
+                First Week
+              </h3>
+                <ul class="languages-grid">                    
+                        <li class="lang-list l1"><strong>Github</strong></li>
+                        <li class="lang-list l2"><strong>Css3</strong></li>
+                        <li class="lang-list l3"><strong>Git</strong></li>
+                        <li class="lang-list l4"><strong>Html5</strong></li>
+                </ul>
+                <div class="external_link">
+                    <a href="#">See Project</a>
+                </div>
+            </div>
+        </div>
+        <div class="project-2">
+            <div class="first_project_img_div">
+            <img src="img/default_project_img.JPG" alt="first project image" class="first_project_image">
+            </div>
+            <div class="first-project">
+              <h3>
+                Setup-Portfolio-Mobile-1st <br>
+                First Week
+              </h3>
+                <ul class="languages-grid">                    
+                        <li class="lang-list l1"><strong>Github</strong></li>
+                        <li class="lang-list l2"><strong>Css3</strong></li>
+                        <li class="lang-list l3"><strong>Git</strong></li>
+                        <li class="lang-list l4"><strong>Html5</strong></li>
+                </ul>
+                <div class="external_link">
+                    <a href="#">See Project</a>
+                </div>
+            </div>
+        </div>
+    </div>       
     </section>
     <section id="about-me">
         <div class="about-me-bg">
-        <h3 class="about-me-tittle">About me</h3>
-        <div class="parraf">
-        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
```

Comment:
- [ ] As shown in the Figma design , the about me section , mostly the part in your portfolio where you listed the language , the skills, and the framework, it could be better if you center them. It will make your design stick to the figma design 💯 

| Your design |  Figma design |
| :---: | :---: |
| ![Capture 2](https://user-images.githubusercontent.com/80458199/228807187-4149f85e-aa63-4efc-9749-4c9fc9874845.PNG) |  ![Capture 1](https://user-images.githubusercontent.com/80458199/228807377-27f5244a-209f-4168-8638-8393d2a4287f.PNG) |